### DISCUSSION TITLE
Recent issue with Glances integration

### DISCUSSION THREAD

**Discussion by Eviljubs** [Lissy93/dashy#1591] [Help and Support]
I had Glances widgets working fine but recently it updated and it hasn't been working.  I thought it might be something in my config file but everything there looks fine as far as I can tell.  However, I did a bit of digging in my logs and found that the url being referenced by dashy's glances widget is http://192.168.1.2:61208/api/3/mem. I tried navigating to the page in my browser but it doesn't give any data.  I then found out that glances has a 4th version of the api (guess that's recent?) -- so if I replace replace the 3 with a 4 like this: http://192.168.1.2:61208/api/4/mem.  I get what looks to be data output.  

I would update the appended url string "/api/3/mem/ however, I'm not sure where Dashy actually stores that.  Any chance the code could be updated relatively quickly?  Seems like it could be an easy find/replace job.

**Comment by CrazyWolf13** [comment]
Hi 
Yeah that's nothing on your end, glances recently updated to api v4, as you noted.

I already submitted a PR which got merged yesterday, just re-pull the latest dashy image from dockerhub and it should all work.
